frontend/src/components/signUpComponents/InputFormContainer.js

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react';
+import '../../pages/SignUp.global.scss';
+import {
+  Button,
+  Box,
+  Layout,
+  Image,
+  Input,
+  Cell,
+  Row,
+  Col,
+  Container,
+  Card,
+  FormField,
+} from 'wix-style-react';
+import { Route } from 'react-router-dom';
+
+const InputFormContainer = ({ onRegister }) => {
```

Comment:
`InputFormContainer` name seems to vague, and it's not really a container-component pattern, what about `SignUpForm` ?

Reply:
fixed